elm.json

```diff
@@ -0,0 +1,19 @@
+{
+    "type": "package",
+    "name": "cuducos/elm-format-number",
+    "summary": "Format numbers as pretty strings",
+    "license": "BSD-3-Clause",
+    "version": "6.0.0",
```

Comment:
Last version was `6.0.0`– shouldn't this be `7.0.0`?

Reply:
When you run ‘elm bump’, it will set it to the appropriate version number (depending on public api changes, if any)